events/git-and-github.md

```diff
@@ -0,0 +1,129 @@
+# Git & Github Workshop
+
+## Aim of the Workshop
+
+At the end of the workshop, make sure that audience are comfortable with the following
```

Comment:
~~At the end of the workshop, make sure that audience are comfortable with the following~~
`The workshop should cover and explain the following:`

Starting with `At the end` seems weird.

Reply:
Ya, valid point. Thanks 4 pointing it out